documentation/src/main/wiki/ScrollModifier.md

```diff
@@ -0,0 +1,49 @@
+#### Click Modifier
```

Comment:
I guess it should be `Scroll Modifier` instead of `Click..`? :)

Reply:
Yes, thanks! Fixed!